docs/architecture/_index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Viam Architecture"
+linkTitle: "Architecture"
+weight: 409
+type: "docs"
+description: "How a machine running on Viam is structured, from on-device to cloud communications."
+imageAlt: "Viam architecture"
+images: ["/viam/machine-components.png"]
+tags: ["components", "services", "communication"]
+menuindent: true
+---
+
```

Comment:
We need some intro here

Reply:
How about an "in this page"?

Reply:
I saw your ticket about adding that more broadly, I don't want to start that for reference pages right now (I don't know if ever tbh, therefore I don't want to start doing it on one page) - it goes out of date too quickly. You can add a normal list without the card.